docs/tutorials/google/floquet_calibration_example.ipynb

```diff
@@ -257,28 +257,28 @@
    "source": [
```

Comment:
<p>Could use <code>create_linear_chain_segment</code>  here instead of <code>create_linear_chain_circuit</code>  now, which might be epsilon more readable. But you'd have to add measurements in <code>create_linear_chain_segment</code>.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4528/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
yeah probably a wash. I wouldn't want to have measurements in the helper function